packages/core/logging/core-logging-server-internal/src/layouts/json_layout.ts

```diff
@@ -43,7 +43,7 @@ export class JsonLayout implements Layout {
     const transactionId = record.meta?.transaction?.id ?? record.transactionId;
 
     const log: Ecs = {
-      ecs: { version: '8.4.0' },
+      ecs: { version: '8.6.0' },
```

Comment:
I think?
```suggestion
      ecs: { version: '8.7.0' },
```

Reply:
I have generated the typings here using 8.6 (the current ecs release), do you think I should go with latest main? That would be `8.7.0-dev` or something like that.

Reply:
hmm... I am not sure, but what I do know that once this is shipped in 8.7 as a user I'd expect it to be `'8.7.0'`. I think have updated to `'8.7.0'` by Feature Freeze, but if it was me I would do it now to get it off my to do list